app/config/schema.json

```diff
@@ -809,6 +809,29 @@
       },
       "additionalProperties": false,
       "required": [ "handlers" ]
+    },
+    "smtp": {
+      "type": "object",
+      "title": "SMTP configuration",
+      "properties": {
+        "host": {
+          "type": "string",
+          "title": "SMTP host",
+          "default": "localhost"
+        },
+        "username": {
+          "type": "string",
+          "title": "SMTP username"
+        },
+        "password": {
+          "type": "string",
+          "title": "SMTP password"
```

Comment:
Could you add required lengths (>8) to username and password (and add dummy values to the test config) to prevent us from accidentally deploying empty values?